client/types/KeyguardRequest.d.ts

```diff
@@ -62,14 +62,16 @@ declare namespace KeyguardRequest {
     type SignTransactionRequest = SimpleRequest & TransactionInfo & {
         layout?: SignTransactionRequestLayout
         shopOrigin?: string
```

Comment:
shopOrigin should not be optional.

Reply:
This is a SignTransaction Request. Not Checkout specifically. I think we should split this Request into two then @sisou?

Reply:
ohhhhh... right! I was confusing it with `appName`. Well, damn :(
I don't even remember why I commented this. Anyway, I looked at the parsing code again and I think it's fine, as you only require `shopOrigin` to be set when the `layout = 'checkout'` anyway.